content/en/apps/reference/app-settings/transitions.md

```diff
@@ -676,58 +675,96 @@ If this configuration is not set then the message defaults to what is set in the
 
 ### create_user_for_contacts
 
+Users are automatically created for certain contacts.  Both creating a new user for a new contact and replacing an existing user with a new user are supported.
+
+#### Configuration
+
+Several configurations are required in `app_settings` to enable the `create_user_for_contacts` transition.
+
+[Login by SMS]({{< ref "apps/reference/api#login-by-sms" >}}) must be enabled by setting the `token_login` configuration.
+
+The [`app_url` property]({{< ref "apps/reference/app-settings#app_settingsjson" >}}) must be set to the URL of the application. This is used to generate the token login link for the new user.
+
+##### Example
+```json
+"app_url": "https://demo.app.medicmobile.org",
+"token_login": {
+  "enabled": true,
+  "translation_key": "sms.token.login.help"
+},
+"transitions": {
+  "create_user_for_contacts": true
+}
+```
+
+#### Create User
+
+When adding a new contact, the `create_user_for_contacts` transition can be triggered to create a new user associated with that contact. _Available since 4.2.x._
+
+##### Example scenario
+
+This functionality would allow a supervisor to add a new contact for a new CHW and have a user for the new CHW automatically be created with the data from the new contact.
```

Comment:
Re-wording for clarity:

```suggestion
Allow a supervisor to onboard a CHW by creating both a new contact and a new user in an existing place with the data from a single "create contact" form.
```

Reply:
I have made some additional tweaks and edits based on your suggestion here.  Let me know if it still sounds confusing!

Reply:
new contents looks great - thanks!